sympy/functions/elementary/hyperbolic.py

```diff
@@ -209,6 +209,52 @@ class cosh(HyperbolicFunction):
     sinh, tanh, acosh
     """
 
+    @property
+    def is_positive(self):
+        from sympy import pi
+        arg = self.args[0]
+
+        if arg.is_real:
+            return True
+
+        re, im = arg.as_real_imag()
+        if im % pi == 0 or re == 0:
```

Comment:
I have not much experience with boolean expressions but maybe something like this would work:
```
x, y = arg.as_real_imag()
yred = y % (2*pi)
return yred == 0 | x == 0 & ((yred < pi/2) | (3*pi/2 < yred))
```
(I'm not sure if there should be additional parentheses.)